client/examples/react/src/App.tsx

```diff
@@ -187,6 +202,10 @@ const App = () => {
           onClick={() => withdrawExample()}
           children="Withdraw (to same address as the ticketer)"
         />
+        <button
+          onClick={() => getVmStateExample()}
+          children="VM State"
+        />
```

Comment:
You don't need to add this button.
But if you want to set the state when you click on this button then you have to do the same as your useEffect